### ISSUE TITLE
Backend 504 errors are not handled properly

### ISSUE THREAD

**Issue by ctrlrsf** [cboscolo/elb2loggly#9]
When an ELB is unable to contact a backend server due to timeout, it logs a 504 in access log with backend field set to a single '-' and not a host:port pair. The code currently splits the backend field by a ':', which increases data.length by 1. If this doesn't occur due to the single hyphen, then data.length does not equal COLUMNS.length, causing error to be logged to console and log record to be skipped.

https://github.com/cboscolo/elb2loggly/blob/master/elb2loggly.js#L74

Sample ELB access log entry showing '-' for backend and 504 HTTP response code:

```
2015-08-03T19:35:21.383076Z xxxxx-production 1.1.1.1:63484 - -1 -1 -1 504 0 0 0 "GET https://xxxx:443/xxxx"
```

CloudWatch log:

```
2015-08-03T19:45:33.221Z    33b9d1ae-3a18-11e5-ba96-ebe34fa0e019    ELB log length 14 did not match COLUMNS length 15
```

**Comment by ctrlrsf** [comment]
@Karnifexx can you review this when you get a chance? I see your PR that was just merged is also affected since you split on ':' in backend field, which can sometimes be '-' if ELB is not able to contact backend.

**Comment by MarcFletcher** [comment]
Yea, should be able to fix that issue. I'll take a look.

**Comment by MarcFletcher** [comment]
Have created PR #10 which addresses this problem.

**Comment by cboscolo** [comment]
Fixed in 31e7c4817a2d78992b6e2a271c2a2c3512d60a70
Thanks @Karnifexx and @ctrlrsf!
